### ISSUE TITLE
UI glitching on Galaxy Note 3

### ISSUE THREAD

**Issue by Eggerd** [localsend/localsend#578]
### Describe the bug

The UI of the Android app is glitching severely on my Samsung Galaxy Note 3. This starts as soon as the app is opened. Although I notice on older versions (1.5.2 | 1.7.0) this only starts once the rotating logo appears.

The glitching can be stopped by navigating to the settings an scrolling, but starts again when interacting with one of the dropdowns, scrolling all the way to the bottom, or when leaving the settings.

### To reproduce

1. Open the app

### Expected behavior

UI should not glitch / flicker the whole screen

### Screenshots

https://github.com/localsend/localsend/assets/904567/7da22182-577b-4d1c-bbf8-6fd7830a76f6

### Smartphone (please complete the following information)

- Device: Samsung Galaxy Note 3
- OS: LineageOS 18.1 (Android 11)
- Version: 1.10.0 | 1.9.1 | 1.9.0 | 1.7.0 | 1.5.2

**Comment by TheGB0077** [comment]
Any other apps using flutter present the same behavior @Eggerd?

**Comment by Eggerd** [comment]
I'm not aware whether any of my other apps use flutter. So I googled for some apps and found [Grey](https://github.com/avirias/Grey) and apparently [eBay](https://flutter.dev/showcase/ebay). Tested both and neither of them show that behavior

**Comment by TheGB0077** [comment]
https://github.com/flutter/flutter/issues/105093
This issue seems to correlate to your problem. The issue is definitively related to the flutter rendering engine and LineageOS.

**Comment by TheGB0077** [comment]
It looks like the problem has been introduced in Flutter 3.0 and affects LineageOS 16 and above.

**Comment by Eggerd** [comment]
You're right, that seems to be the core issue. Thanks for pointing that out
